app/views/external_users/claims/new.html.haml

```diff
@@ -1,3 +1,10 @@
-= render partial: 'layouts/header', locals: {page_heading: t('.heading')}
+%header.main-header{role: 'banner'}
```

Comment:
I dont think this should be in here because the layout/header partial will output similar markup so the page would have two main headings.